lib/helpers.js

```diff
@@ -524,3 +545,7 @@ exports.getBluemixDefaultPlan = function(serviceType) {
     default:                    return 'Lite';
   }
 }
+
+exports.ignoreFile = function(ignoreLine) {
```

Comment:
hmm ok, what exactly is the difference between `fs` and `this.fs`? Should I keep this `ingnoreFile` function in the helpers.js file?